<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetSite</class>
 <widget class="QDialog" name="SetSite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>站点</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/language-black-48dp.svg</normaloff>icon/language-black-48dp.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAddress">
       <property name="text">
        <string>地址</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAddress">
       <property name="text">
        <string>http://swu.edu.cn</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelComment">
       <property name="text">
        <string>备注</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="plainTextComment"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditTitle"/>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetSite</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetSite</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
